packages/css/src/components/justify-form-groups/justify-form-groups.template.ts

```diff
@@ -0,0 +1,45 @@
+import { html } from 'lit-html';
+
+import { formGroupSelectTemplate } from '../form/form-groups/select/form-group-select.template';
+import { formGroupInputTemplate } from '../form/form-groups/input/form-group-input.template';
+import { buttonTemplate } from '../button/button.template';
+
+export function justifyFormGroupsTemplate() {
```

Comment:
Deze template function is niet generiek, er zit al teveel in. Daarom kunnen wij (als maintainers) dit template niet meer herbruiken. Graag alles verplaatsen naar modelering, zie opmerking over modelering in het issue.